doc/adr/0007-bundesident-url-scheme.md

```diff
@@ -0,0 +1,29 @@
+# 7. Switch from eid:// to bundesident:// scheme
+
+Date: 2022-12-13
+
+## Status
+
+Accepted
+
+## Context
+
+The Technical Guideline TR-03124-1 eID-Client – Part 1 (version 1.4 from 8. October 2021, Chapter 2.2 Full eID-Client) specifies the URL scheme for the ClientURL. 
+Since we are implementing a mobile only approach, described in ADR 5, we need to define the ClientURL as follows: 'eid://127.0.0.1:24727/eID-Client'. 
+However, our eID-Client should not serve as a replacement of the currently available eID-Clients, e.g. AusweisApp2 (AA2). 
+Our ClientURL schema should only open our app (BundesIdent) and should not interfere with other eID-Clients. 
+
+## Decision
+
+Instead of <br>
+'eid://127.0.0.1:24727/eID-Client' <br>
+we use <br>
+'**bundesident**://127.0.0.1:24727/eID-Client' <br>
+
+as our ClientURL schema. 
```

Comment:
In my opinion, I'd remove the newlines and use `code` block instead of `'`.

And you could add a sentence, that we use that as fallback only and use Universal Links/App Links as default, referencing to [0013 Format of the eID-Client URL](https://github.com/digitalservicebund/useid-backend-service/blob/main/doc/adr/0013-format-of-eid-client-url.md).

Reply:
this section currently looks like this below

![image](https://user-images.githubusercontent.com/9930966/207610695-bb0d878d-31ac-4e29-9214-b41518234063.png)

do you mean you want to use the backtick (hightlight as code)?

Instead of
`eid://127.0.0.1:24727/eID-Client`
we use 
`bundesident://127.0.0.1:24727/eID-Client`